taskboot/github.py

```diff
@@ -130,3 +131,47 @@ def github_release(target, args):
         release.upload_asset(name=asset_name, path=artifact_path, label=asset_name)
 
     logger.info(f"Release available as {release.html_url}")
+
+
+def github_push(target, args):
+    """
+    Push commits on a GitHub repository
+    """
+
+    # Load config from file/secret
+    config = Configuration(args)
+    assert config.has_github_auth(), "Missing GitHub authentication"
+
+    # Setup GitHub API client
+    github = Github(config.github["token"])
+
+    # Get user
+    user = github.get_user()
+
+    # Set remote repository
+    try:
+        repo_link = (
+            "https://"
+            + user.login
+            + ":"
+            + config.github["token"]
+            + "@github.com/"
+            + args.repository
+            + ".git"
+        )
+        subprocess.run(["git", "remote", "set-url", "origin", repo_link], check=True)
+
+    except subprocess.CalledProcessError:
+        raise Exception(f"Adding {args.repository} as a remote repository failed")
+
+    # Push on repository
+    try:
+        if args.force_push:
+            push_command = "git push -f origin " + args.branch
+        else:
+            push_command = "git push origin " + args.branch
+
+        subprocess.run(push_command.split(), check=True)
```

Comment:
Nit: just make `push_command` a list directly instead of making it a string and then splitting it into a list